### DISCUSSION TITLE
RSE Silos

### DISCUSSION THREAD

**Discussion by simoncstanton** [rse-aunz/organisation#53] [RSEAA2022]
Knowledge is often silo-ed - so much so that even disciplines with overlapping ares of interest can separately 're-invent the wheel', so to speak. What are some good examples of this that you have come across in your experience? How would you even know if another field has also implemented a solution that your field has? How could we increase transparency or promote visibility so that we can learn from each other and not have to lose time, and resources?

**Comment by robmoss** [comment]
One reason that this can be really hard is that different research fields/disciplines use different terminology to describe the same thing, and it can require an in-depth conversation to realise that you and the other person are talking about the same idea/concept/method.

**Comment by nrmay** [comment]
Could this be supported by a web platform such as [A Web Platform for RSEs](https://github.com/orgs/rse-aunz/discussions/56)?
